src/validator/validator.ts

```diff
@@ -59,17 +61,21 @@ class Validator {
   }
 
   /**
-   * Creates a new block proccessing service and starts it.
+   * Creates a new block processing service and starts it.
    */
-  private async start(): Promise<void> {
+  public async start(): Promise<void> {
+    this.isRunning = true;
     await this.setup();
-    this.run();
+    // this.run();
```

Comment:
this should not be commented

Reply:
Sorry, had commented it for testing. Forgot to uncomment.